code/game/machinery/firealarm.dm

```diff
@@ -55,6 +55,9 @@
 					set_light(l_range = L_WALLMOUNT_RANGE, l_power = L_WALLMOUNT_POWER, l_color = "#1024A9")
 				if("red")
 					previous_state = icon_state
+					set_light(l_range = L_WALLMOUNT_HI_RANGE, l_power = L_WALLMOUNT_HI_POWER, l_color = COLOR_YELLOW)
```

Comment:
These should probably be using the `LIGHT_COLOR_*` defines, not the `COLOR_*` defines.